pom.xml

```diff
@@ -45,7 +45,7 @@
     <properties>
         <oak.version>1.44.0</oak.version>
```

Comment:
These versions of Oak and Jackrabbit are EOLd.

Please use the latest maintenance branch versions (Jackrabbit 2.20.15 and Oak 1.22.19).

Reply:
I think this change would be better done in a separate PR, with a related Jira ticket since it's unrelated to the goal of this ticket and PR. I would rather someone else did it, someone who understands the implications.

Reply:
i've created [SLING-12287](https://issues.apache.org/jira/browse/SLING-12287) for the oak version discussion